ynput/OpenPype
# Enhancement kitsu note with exceptions

## Description

Adding a setting to choose some exceptions to IntegrateKitsuNote task status changes.

## Testing

- Go to project settings -> Kitsu -> Publish.
- With integrate kitsu note active, add entries to the exception list and set them up.
- Go on kitsu and set the status of the task you want to use for testing, so the exceptions would apply.
- Publish this task.
- The task status should remain unchanged.
- You can also test that in a case in which the exceptions should not trigger, then the status set as `Note Status Shortname` is applied.
